src/org/zaproxy/zap/extension/websocket/db/TableWebSocket.java

```diff
@@ -583,6 +583,7 @@ public void insertOrUpdateChannel(WebSocketChannelDTO channel) throws DatabaseEx
 			
 					stmt.setString(1, channel.host);
 					stmt.setInt(2, channel.port);
+					logger.debug("url (length " + channel.url.length() + "):" + channel.url);
```

Comment:
I was concerned about log burden and wrapped it in logger.debug for this reason. Per [Log4J intro](http://logging.apache.org/log4j/1.2/manual.html): "When logging is turned off entirely or just for a set of levels, the cost of a log request consists of a method invocation plus an integer comparison." Does this address your concern?